SourceX/qol.cpp

```diff
@@ -10,10 +10,25 @@
 namespace devilution {
 namespace {
 
+namespace xpbar {
+	constexpr int barWidth = 307;
+	constexpr int barHeight = 5;
+
+	constexpr int goldGradient[] = { 0xCF, 0xCE, 0xCD, 0xCC, 0xCB, 0xCA, 0xC9, 0xC8, 0xC7, 0xC6, 0xC5, 0xC4, 0xC3, 0xC2, 0xC1, 0xC0 };
+	constexpr int goldGrades = 12;
+
+	constexpr int whiteGradient[] = { 0xFE, 0xFD, 0xFC, 0xFB, 0xFA, 0xF9, 0xF8, 0xF7, 0xF6, 0xF5, 0xF4, 0xF3, 0xF2, 0xF1, 0xF0, 0xFF };

```

Comment:
`[0] = 0xFE`, `[15] = 0xFF` — is this intentional? `goldGradient` has a different order.

Reply:
Yes, intentional. The black/white gradient has unexpected order.

Compare the top row (greyscale gradient) in this screenshot to the other 16 color gradients (4th row is the gold gradient):
![image](https://user-images.githubusercontent.com/81810641/114952391-74453200-9e56-11eb-972e-ceae06114457.png)